src/components/Question.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default class Question extends React.Component {
+  render() {
+    const questions = ['How would you want to spend your vacation?',
```

Comment:
Will this be ever server side rendered? i.e. Fetch these from the server instead of hard-coding it here